### ISSUE TITLE
Slowing down

### ISSUE THREAD

**Issue by notnot-nick** [tsparticles/tsparticles#1414]
**Which version are you using?**

1.24.2

**Which library are you using?**

- [ x ] React (`react-tsparticles`)

<details>
 <summary><b>tsParticles Configuration</b></summary>
  
  ```javascript
{
				fpsLimit: 60,
				particles: {
					color: {
						value: '#ffffff',
					},
					links: {
						color: '#ffffff',
						distance: 150,
						enable: true,
						opacity: 0.5,
						width: 1,
					},
					collisions: {
						enable: true,
					},
					move: {
						direction: 'none',
						enable: true,
						outMode: 'bounce',
						random: true,
						speed: 3,
						straight: false,
					},
					number: {
						density: {
							enable: true,
							value_area: 800,
						},
						value: 80,
					},
					opacity: {
						value: 0.5,
					},
					shape: {
						type: 'circle',
					},
					size: {
						random: true,
						value: 5,
					},
				},
				detectRetina: true,
			}
  ```
</details>

**Describe the bug**

Particles end up getting REALLY SLOW, like almost not moving after a while

**To Reproduce**
1. Create a react project with tsparticles and use my config

**Expected behavior**

Particles keep going brrrr

**Screenshots**

After reloading, particles get fast again

https://owfa.host/2IyWiQ

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Firefox
 - Version: 86.0.1

**Comment by matteobruni** [comment]
Thanks for reporting that @Soldr, it seems related to `OutMode.bounce`, I'll investigate further.

**Comment by leoseccia** [comment]
I am experiencing the same - Ubuntu 20.10 / Chrome 89 (64 bit)

**Comment by matteobruni** [comment]
I found the cause, every time two particles collide they slow down.

I don't think this is an issue since it's a normal behavior. Anyway you can override the bounce values between particles.

I created a sample with CodeSandbox here: <https://codesandbox.io/s/weathered-tdd-09zhl?file=/src/App.js>

In this sample you can see how to customize particles bounces factors. They are multipliers to horizontal and vertical velocities so be careful with values.

**Comment by matteobruni** [comment]
I'm currently digging more deeply since I'm not 100% sure about what I said. If I found a bug I'll let you know.

**Comment by matteobruni** [comment]
Ok with a  different mass calculation, using the particle area instead of radius, the effect seems nicer, but I'll do more tests before releasing an update.

**Comment by matteobruni** [comment]
With the new version released (`1.24.2` core, `1.24.3` react) this issue is fixed, I changed the formula using a different mass and it looks good.

**Comment by leoseccia** [comment]
Nice one! I will upgrade now :-) Thank you.
